Android/3D Cave Runner - Android Studio/local.properties

```diff
@@ -4,6 +4,6 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Wed Aug 15 17:34:46 PDT 2018
-ndk.dir=/Users/gherlihy/Library/Android/sdk/ndk-bundle
-sdk.dir=/Users/gherlihy/Library/Android/sdk
+#Thu Jun 13 14:08:57 PDT 2019
+ndk.dir=/Users/emily/Library/Android/sdk/ndk-bundle
+sdk.dir=/Users/emily/Library/Android/sdk
```

Comment:
Personally, I prefer `gherlihy` here. Didn't we add this file to our `.gitignore` file above?

Reply:
i just added it in the second commit in this pr, so i guess its going to be `emily` for the rest of time 💁‍♀️